test/DotNetWorkerTests/AspNetCore/FunctionsHttpProxyingMiddlewareTests.cs

```diff
@@ -54,7 +54,12 @@ private static (FunctionContext FunctionContext, HttpContext HttpContext, Mock<I
                 { "test", new TestBindingMetadata("test", triggerType, BindingDirection.In ) }
             };
 
-            var functionDef = new TestFunctionDefinition(inputBindings: inputBindings);
+            var outputBindings = new Dictionary<string, BindingMetadata>()
+            {
+                { "outputBinding", new TestBindingMetadata("$return", "http", BindingDirection.Out ) }
+            };
+
+            var functionDef = new TestFunctionDefinition(inputBindings: inputBindings, outputBindings: outputBindings);
 
             var functionContext = new TestFunctionContext(functionDef, new TestFunctionInvocation(), CancellationToken.None)
```

Comment:
Not seeing tests for the new behavior added to the proxying middleware. Can we please have those added?